app/static/js/slcManagement.js

```diff
@@ -25,8 +25,50 @@ $(document).ready(function() {
       }
     }
   });
-});
+  const emailSelectedButton = $("#emailSelectedButton");
+  const selectAllOthersButton = $("#selectAllOthersButton");
+  const instructorCheckboxes = $(".instructorCheckbox");
+
+  emailSelectedButton.on("click", function () {
+    const selectedEmails = Array.from(instructorCheckboxes)
+      .filter((checkbox) => checkbox.checked)
+      .map((checkbox) => checkbox.getAttribute("data-email"))
+      .join(";");
```

Comment:
line filter, map, and join up under from

Reply:
snap, crackle, pop, hook line and sinker lights camera action